apps/site/assets/ts/schedule/__tests__/LineDiagramTest.tsx

```diff
@@ -537,14 +541,60 @@ export const lineDiagram = [
   }
 ] as LineDiagramStop[];
 
+const stops = lineDiagram.map(({ route_stop }) => ({
```

Comment:
It might be more intention-revealing to call this `simpleStops`, since it matches the shape of a `SimpleStop`. (or, could we typecast it as `SimpleStop[]`?)